http://bugs.kde.org/show_bug.cgi?id=131987

--- akode/plugins/ffmpeg_decoder/ffmpeg_decoder.cpp     2006-08-07 06:26:10.000000000 +0500 
+++ akode/plugins/ffmpeg_decoder/ffmpeg_decoder.cpp.new 2006-08-07 06:29:24.000000000 +0500 
@@ -117,18 +117,21 @@
     if (config->channels > 2) return false;
     config->channel_config = MonoStereo;
     switch(codec_context->sample_fmt) {
+        case SAMPLE_FMT_U8: 
+            config->sample_width = 8; 
+            break; 
         case SAMPLE_FMT_S16:
             config->sample_width = 16;
             break;
+        case SAMPLE_FMT_S24: 
+            config->sample_width = 24; 
+            break; 
         case SAMPLE_FMT_S32:
             config->sample_width = 32;
             break;
         case SAMPLE_FMT_FLT:
             config->sample_width = -32;
             break;
-        case SAMPLE_FMT_DBL:
-            config->sample_width = -64;
-            break;
         default:
             return false;;
      }
